src/coreclr/jit/emitxarch.h

```diff
@@ -160,6 +168,249 @@ code_t AddVexPrefixIfNeededAndNotPresent(instruction ins, code_t code, emitAttr
     return code;
 }
 
+// TODO-XArch-AVX512: Explore adding this as a flag to instr table.
```

Comment:
👍, using a flag would definitely be a net positive IMO

Reply:
Do I leave the comment for now and circle back with the instr table change as a later update or would you prefer folding instr table change to this PR?

Reply:
I think its fine to do it separately to help minimize total churn